### ISSUE TITLE
tr and td tags removed on Windows 8.1

### ISSUE THREAD

**Issue by sebarslan** [MSOpenTech/winstore-jscompat#25]
Hi,
kendo ui grid does not display correctly in Windows 8.1

tr and td tags is removed on Windows 8.1.

Windows 8.1
[http://prntscr.com/9wav54](url)

Windows 10
[http://prntscr.com/9wavmt](url)

thanks

**Comment by montella1507** [comment]
Yes, we have the same problem in every application.. i hope it will be solved :-( 

http://imgur.com/7DP8olq

**Comment by sebaarslan** [comment]
I think, today I've found for kendo grid a solution and it works. 

Change the function
function insertAdjacentHTML(element, position, html) {
    HTMLElement_insertAdjacentHTMLPropertyDescriptor.value.call(element, position, html);
 }

to

function insertAdjacentHTML(element, position, html) {
    if(element.indexOf("<tbody") !== 0 && html.indexOf("<tr") === 0)
    {
        element = '<tbody role="rowgroup"></tbody>';
    }
    HTMLElement_insertAdjacentHTMLPropertyDescriptor.value.call(element, position, html);
 }

**Comment by sebarslan** [comment]
I think, today I've found for kendo grid a solution and it works.

Change the function

```
function insertAdjacentHTML(element, position, html) {
HTMLElement_insertAdjacentHTMLPropertyDescriptor.value.call(element, position, html);
}
```

to

```
function insertAdjacentHTML(element, position, html) {
if(html.indexOf("<tr") === 0 && element.indexOf("<tbody") !== 0 ) {
element = '<tbody role="rowgroup"></tbody>';
}
HTMLElement_insertAdjacentHTMLPropertyDescriptor.value.call(element, position, html);
}
```

**Comment by montella1507** [comment]
Great i will try.. but this is propably the solution only for one thing (Kendo grid) and we have to count with problems with other components too :-( So it would be nice to have common fix for these issues..

**Comment by sebarslan** [comment]
Yes. you're right.

**Comment by montella1507** [comment]
Atleast,.. good work, it works for me.. how did u debug the compat-js code? I am using Visual Studio to package Apache cordova app and i think i am not able to debug "native" js code

**Comment by sebarslan** [comment]
I also use cordova javasript app and debug mode works. But I do not know, if you can debug the native code.
